cirq-google/cirq_google/ops/internal_gate_test.py

```diff
@@ -67,3 +71,74 @@ def test_internal_gate_with_hashable_args_is_hashable():
     )
     with pytest.raises(TypeError, match="unhashable"):
         _ = hash(unhashable)
+
+
+def test_internal_gate_with_custom_function_repr():
+    original_func = lambda x: x**2
+    x = np.linspace(-1, 1, 10)
+    y = original_func(x)
```

Comment:
nit - original_func is not used
```suggestion
    y = x ** 2
```